sql.txt

```diff
@@ -1,14 +1,32 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
-Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
+Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below.
+
+SELECT CustomerName, ContactName, Country FROM Customers ORDER BY Country LIMIT 10 OFFSET 10;
+
 
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
+SELECT * FROM Customers WHERE ContactName LIKE "A%";
+
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
+SELECT * FROM OrderDetails WHERE ProductID = 51 AND quantity > 10;
+
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
 
+INSERT INTO Products (ProductID, ProductName, SupplierID, CategoryID, Unit, Price) Values
+       	    	     		 	      	(NULL, "Bobsled", 5, 2, "1 box", 300),
+						(NULL, "Trampoline", 4, 1, "2boxes",90),
+						(NULL, "Knife", 7, 3, "1 box", 30);
+
+
 (5) Update the two top rows of the Products Table to increase the price by 1.50. (Get SQL to do the addition for you.)  Paste your SQL statement below.
 
+UPDATE products SET price = price + 1.50 WHERE ProductID IN (1, 2);
```

Comment:
Nice job here, when using mathematical operations, personally I like to do `(price + 1.50)` to clearly separate it from another statement.